app/front/scripts/services/download-package.js

```diff
@@ -80,8 +80,8 @@
               StorageService.clearApplicationState()
                   .then(function() {
                     var packageName = PackageService.getAttributes().name;
-                    var owner = LoginService.email;
-                    state.packagePublicUrl = '/viewer/'+owner+':'+packageName; //dataPackage.uploadUrl;
+                    var owner = LoginService.userid;
```

Comment:
@akariv a question: as we accept other auth providers in future, won't this break? eg: Paul logged in with Google and Paul logged in with Facebook will be different "owners".

So, shouldn't we be using our own uuid or similar that is consistent for a user, no matter which auth backend?